pkg/handler/image/image.go

```diff
@@ -60,10 +82,263 @@ func (c *Client) ListImages(ctx context.Context, organizationID, regionID string
 		return nil, errors.OAuth2ServerError("failed to list images").WithError(err)
 	}
 
+	if filterFn != nil {
+		result = slices.Clone(result)
+		result = slices.DeleteFunc(result, filterFn)
+	}
+
 	// Apply ordering guarantees, ordered by name.
 	slices.SortStableFunc(result, func(a, b types.Image) int {
 		return cmp.Compare(a.Name, b.Name)
 	})
 
 	return convertImages(result), nil
 }
+
+func (c *Client) ListActiveImages(ctx context.Context, organizationID, regionID string) (openapi.Images, error) {
```

Comment:
This appears unused... I think in future a user would want to see an image that's uploading (as extra feedback it's actually doing something) so they can poll for completion.  🤷🏻